model/connect/PDOConnector.php

```diff
@@ -178,7 +179,7 @@ public function connect($parameters, $selectDB = false) {
 	}
 
 	public function getVersion() {
-		return $this->pdoConnection->getAttribute(PDO::ATTR_SERVER_VERSION);
+		return @$this->pdoConnection->getAttribute(PDO::ATTR_SERVER_VERSION);
```

Comment:
"Supress warnings on getting the server version, attributes aren't supported by the dblib driver"